moped-editor/src/views/projects/projectView/SignalComponentAutocomplete.js

```diff
@@ -1,60 +1,19 @@
 import React from "react";
-import { TextField, CircularProgress } from "@material-ui/core";
+import { useState } from "react";
+import { CircularProgress } from "@material-ui/core";
 import { Autocomplete, Alert } from "@material-ui/lab";
-import { useEffect, useState } from "react";
 import { useSocrataGeojson } from "src/utils/socrataHelpers";
-import { signalToFeatureCollection } from "src/utils/mapHelpers";
-
+import {
+  filterSignalOptions,
+  useSignalChangeEffect,
+  getSignalOptionLabel,
+  getSignalOptionSelected,
+  useInitialSignalComponentValue,
+  renderSignalInput,
+} from "src/utils/signalComponentHelpers";
```

Comment:
Unified autocomplete behavior between the two versions of the component (new project vs editing).